locales/fr/common.json

```diff
@@ -1,12 +1,11 @@
 {
-    "home": "Accueil",
-    "about": "A propos",
-    "faq": "FAQ",
-    "media": "Demandes des Médias",
-    "add_story": "Ajoutez votre Histoire",
-    "analytics": "Analytique",
-    "email": "Envoyez-nous un Email",
-    "translate": "Aidez-nous à Traduire",
-    "made_with": "Fabriqué avec amour par les Ontariens ❤️",
-    "tagline": "Si nos dirigeants n’écoutent pas les chiffres, ils doivent faire face à nos histoires."
-}
\ No newline at end of file
+  "home": "Accueil",
+  "about": "À propos",
+  "faq": "FAQ",
+  "media": "Médias",
+  "add_story": "Ajoutez votre histoire",
+  "analytics": "Analytique",
+  "email": "Envoyez-nous un email",
+  "translate": "Aidez-nous à traduire",
```

Comment:
I noticed that most of your French translations use sentence case (not title case), which I think is right. I changed the couple that didn't.